src/hlstrafe.cpp

```diff
@@ -2018,6 +2054,9 @@ namespace HLStrafe
 				case HLTAS::ConstraintsType::YAW_RANGE:
 					constraints = 0.1;
 					break;
+				case HLTAS::ConstraintsType::LOOK_AT:
+					constraints = 0.1;
```

Comment:
As far as I can tell, this should be 0 and YAW_RANGE should also be 0 (please change it). Maybe I wrote this code before constraints = 0 was efficient.